curtsies/escseqparse.py

```diff
@@ -66,7 +66,7 @@ def peel_off_esc_code(s):
         del d['front']
         del d['rest']
```

Comment:
What is the purpose of these `del`s when we're returning `m.groupdict()['front']` & rest on line #71 below?